frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -41,6 +41,9 @@ export class CatalogTileDetails extends React.Component {
       documentationUrl,
       sampleRepo,
     } = this.props.item;
+    const {
+      'marketplace.openshift.io/support-workflow': marketplaceSupportWorkflow,
+    } = this.props.item.obj.csv.metadata.annotations;
```

Comment:
We should just set this as `supportUrl` when the catalog item is created

Reply:
This should be set as `supportUrl` here:

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts#L70

Reply:
Thanks!